group_vars/sn06.yml

```diff
@@ -242,6 +242,10 @@ pip_install_dependencies:
   # used by reports
   - WeasyPrint
   - total_perspective_vortex==2.0.1
+  # Needed for usegalaxy-eu.vgcn-monitoring Telegraf role
+  - pyyaml
+  - GitPython
+  - python-openstackclient
```

Comment:
> Have you checked the dependencies for possible conflicts? (e.g. with [pipdeptree](https://pypi.org/project/pipdeptree/))

There are no conflicts. 

> Shouldn't this run on the monitoring VM only? I thought the monitoring VM builds the venv and syncs it to NFS

The maintenance node is currently not active (since this [PR](https://github.com/usegalaxy-eu/infrastructure/pull/147) merged) and is also not in production. When it moves to production, we will clean up sn06 and sn07 playbooks and group vars files, remove redundant things and use this [document](https://github.com/usegalaxy-eu/operations/blob/main/head_maintenance_nodes.md) to separate stuff. Until then this role will run on sn06 (not in sn07, I will push a new commit that will remove the role from the sn07 playbook) so we can proceed further with the development of the Grafana dashboard

Reply:
I have [removed this role from sn06](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/630/commits/ea75492c49b3fd93ef383d10d71c0d4ce194ab30) and it will be directly deployed on the maintenance node.